scripts/tar_web_vault.sh

```diff
@@ -18,13 +18,21 @@ mkdir -pv "${BASEDIR}/../${OUTPUT_FOLDER}"
 pushd "${VAULT_FOLDER}/apps/web"
 
 VAULT_VERSION=$(get_web_vault_version)
-OUTPUT_NAME="${OUTPUT_FOLDER}/bw_web_${VAULT_VERSION}"
+OUTPUT_NAME="${BASEDIR}/../${OUTPUT_FOLDER}/bw_web_${VAULT_VERSION}"
+DATE_FORMAT="${DATE_FORMAT:-%Y-%m-%dT%H:%M:%S%z}"
 
 mv build web-vault
 # Tar the web-vault
-tar -czvf "${BASEDIR}/../${OUTPUT_NAME}.tar.gz" web-vault --owner=0 --group=0
+tar -czvf "${OUTPUT_NAME}.tar.gz" web-vault --owner=0 --group=0
+
+# Preserve previous output directory
+if [[ -e ${OUTPUT_NAME} ]];
```

Comment:
```suggestion
if [[ -d ${OUTPUT_NAME} ]];
```
I think using `-d` is better then `-e`, since it is more specific.